src/main/java/sandbox/SqlSandboxUtils.java

```diff
@@ -51,7 +51,7 @@ public static SQLiteConnectionPoolDataSource getDataSource(String databaseName,
             }
             Class.forName("org.sqlite.JDBC");
             SQLiteConnectionPoolDataSource dataSource = new SQLiteConnectionPoolDataSource();
-            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db");
+            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db?journal_mode=MEMORY");
```

Comment:
nice. I agree with this logic a lot. 

Since there are plenty of caching related reasons why we'd rebuild a db entirely anyway, seems totally fine to optimize for memory as long as the hard fallback is consistent